x-pack/plugins/snapshot_restore/public/application/sections/home/policy_list/policy_list.tsx

```diff
@@ -8,13 +8,7 @@
 import React, { Fragment, useEffect } from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { RouteComponentProps } from 'react-router-dom';
-import {
-  EuiPageContent_Deprecated as EuiPageContent,
-  EuiEmptyPrompt,
-  EuiButton,
-  EuiCallOut,
-  EuiSpacer,
-} from '@elastic/eui';
+import { EuiPageSection, EuiEmptyPrompt, EuiButton, EuiCallOut, EuiSpacer, EuiPageTemplate } from '@elastic/eui';
```

Comment:
Please remove unused imports

```suggestion
import { EuiButton, EuiCallOut, EuiSpacer, EuiPageTemplate } from '@elastic/eui';
```